di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/config/CredentialIssuerConfig.java

```diff
@@ -1,20 +1,43 @@
 package uk.gov.di.ipv.stub.cred.config;
 
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import java.lang.reflect.Type;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.Map;
+
 public class CredentialIssuerConfig {
     public static final String PORT = getConfigValue("CREDENTIAL_ISSUER_PORT","8084");
     public static final String NAME = getConfigValue("CREDENTIAL_ISSUER_NAME","Credential Issuer Stub");
-    public static final String CLIENT_CONFIG = getConfigValue("CLIENT_CONFIG", null);
 
     public static final String EVIDENCE_STRENGTH_PARAM = "strength";
     public static final String EVIDENCE_VALIDITY_PARAM = "validity";
     public static final String ACTIVITY_PARAM = "activity";
     public static final String FRAUD_PARAM = "fraud";
     public static final String VERIFICATION_PARAM = "verification";
 
+    public static Map<String, ClientConfig> CLIENT_CONFIGS;
```

Comment:
Is there a reason we can't set a value this straight away? As at the moment it will work fine when we are reading this value we are dependant that getClientConfig(clientId) has been called from where for it to have a value, so if we ever do any refactors etc we could accidentally end up trying to access this var before that method call and get an incorrect scenario where it looks like the client configs aren't setup?

Could this be moved below line 25 to be something like 
```suggestion
    public static Map<String, ClientConfig> CLIENT_CONFIGS = parseClientConfigs();
```

Reply:
I did this because it makes the tests work! There might be a better way to do it, but I'm tempted to leave it for now.